### ISSUE TITLE
Stockroom Book + Enhanced Inventories chest = crash

### ISSUE THREAD

**Issue by gchpaco** [Nuchaz/BiblioCraft#321]
Fortunately doesn't bring the server down.  Crash log, such as it is, is this:
```
[00:13:24] [Server thread/WARN]: Failed to handle packet for /75.25.121.2:59059
java.lang.IllegalArgumentException: Empty string not allowed
        at net.minecraft.nbt.NBTTagString.<init>(NBTTagString.java:23) ~[dx.class:?]
        at net.minecraft.nbt.NBTTagCompound.func_74778_a(NBTTagCompound.java:105) ~[dh.class:?]
        at jds.bibliocraft.items.ItemStockroomCatalog.func_77648_a(ItemStockroomCatalog.java:154) ~[ItemStockroomCatalog.class:?]
        at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:505) ~[ForgeHooks.class:?]
        at net.minecraft.item.ItemStack.func_77943_a(Unknown Source) ~[add.class:?]
        at net.minecraft.server.management.ItemInWorldManager.func_73078_a(Unknown Source) ~[mx.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556) ~[nh.class:?]
        at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60) ~[jo.class:?]
        at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9) ~[jo.class:?]
        at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212) ~[ej.class:?]
        at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165) [nc.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334) [lt.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685) [li.class:?]
```
Tested with double and single chests.

**Comment by Nuchaz** [comment]
Hmmm, that shouldn't be happening. I'll look into this when I get some time.

**Comment by Nuchaz** [comment]
I'll  have a fix for this in an upcoming update, v1.12.0
